### ISSUE TITLE
Hexal wisp will not use Greater Sentinel range when summoned.

### ISSUE THREAD

**Issue by qwerty121123** [Talia-12/Hexal#6]
The intended function of the below spell is to fling a `projectile wisp`, and if it hits a living entity, that entity and all other entities within the 15 block range of the `greater sentinel` with lightning, therefore bypassing the `projectile wisp`'s range limitation of 4 blocks.
```
[
  Bookkeeper's Gambit: v-
  Compass' Purification
  Summon Greater Sentinel
  Locate Sentinel
  Numerical Reflection: 15
  Zone Distillation: Living
  {
    Compass' Purification
    Summon Lightning
  }
  Jester's Gambit
  Thoth's Gambit
  Banish Sentinel
]
```
However,
It uses the range of the wisp (4 Blocks) instead of functioning as intended.
_(Off topic but are there any plans to add a delay pattern?)_
Log Files: [pastebin.com](https://pastebin.com/hyq1uc8R)
System Info:

_OS Name	Microsoft Windows 10 Home
Version	10.0.19044 Build 19044
Other OS Description 	Not Available
OS Manufacturer	Microsoft Corporation
System Manufacturer	Dell Inc.
System Model	OptiPlex 3010
System Type	x64-based PC
System SKU	
Processor	Intel(R) Core(TM) i5-3470 CPU @ 3.20GHz, 3201 Mhz, 4 Core(s), 4 Logical Processor(s)
BIOS Version/Date	Dell Inc. A17, 11/16/2016
SMBIOS Version	2.7
Embedded Controller Version	255.255
BIOS Mode	Legacy
BaseBoard Manufacturer	Dell Inc.
BaseBoard Product	042P49
BaseBoard Version	A01
Platform Role	Desktop
Secure Boot State	Unsupported
PCR7 Configuration	Binding Not Possible
Windows Directory	C:\WINDOWS
System Directory	C:\WINDOWS\system32
Boot Device	\Device\HarddiskVolume1
Locale	United States

Time Zone	Atlantic Standard Time_

**Comment by Talia-12** [comment]
This should be fixed now in release v0.1.12 / v0.2.1

**Comment by qwerty121123** [comment]
Thank you so much for backporting the fix to 1.18.2!
